index.js

```diff
@@ -87,31 +79,19 @@ module.exports = Metrics = {
     )
   },
 
-  buildPromKey(key) {
-    if (key == null) {
-      key = ''
-    }
+  buildPromKey(key = '') {
```

Comment:
It does not make sense to track a metric without a key. I'd lean towards removing the default here.